### DISCUSSION TITLE
Dark Mode Hex Value

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#3004] [Q&A]
I'm using a service called Cloudinary and I would like to set the background color of an image to be identical to the dark mode theme background, like so
`<Image publicId={profilePicture} width="150" height="150" radius="70">` 
        `          <Transformation `
         `          background="" Hex code for dark theme would go here` 
          `         gravity="center"`
          `height="200"`
          `radius="max"`
          `width="200"`
          `crop="fill"`
        `/>`
      `</Image>`
`
In other words, how can I retrieve the hex code value for the 'chakra-ui-color-mode' property stored in Local Storage?

**Comment by ghost** [comment]
I figured out an alternative. I have decided to generate the Cloudinary URL source and use Chakra UI's Image component instead, like so
` <Image
          w={100}
          h={75}
          borderRadius={"70%"}
          src={"https://res.cloudinary.com/[my_cloudinary_database]/image/upload/" + picture}
        />`
Picture is the part of url for specific user of a default picture.
